### ISSUE TITLE
Set different learning rate

### ISSUE THREAD

**Issue by Presleyliao** [KaiyangZhou/Dassl.pytorch#63]
Thank you for your great work!

I have a problem would like to ask.
Is it possible to give two different parts in one model with different learning rates?
